### DISCUSSION TITLE
UBL Information

### DISCUSSION THREAD

**Discussion by gmanc2** [Jyers/Marlin#634] [Q&A] [answered]
I was interested in using the 15x15 UBL HS firmware. I was using the 5x5-v4.2.2 previously and wanted to switch to the 15x15 for a more precise automatic leveling mesh. I have a BLTouch and a 4.2.2 board. Is there anything I would need to change if switching between the two? I already set my probe X-Offest to -45 and my Z-Offset to -2.12 but is there anything else I'll have to know before switching over?

**Comment by Jyers** [comment, marked as answer]
Check out the UBL section here: https://github.com/Jyers/Marlin/wiki/Configuration-Details The only changes you really need to make are with your start gcode
